source/vault/ec2.go

```diff
@@ -14,16 +12,3 @@ func getNonce(path string) (string, error) {
 	}
 	return string(bytes.TrimSpace(content)), nil
 }
-
-func getPkcs7() (string, error) {
```

Comment:
Since #6, I think the `getNonce()` function is no longer used either.
Could you double check and remove it if it's not used anymore?

Reply:
I've checked through the entire codebase, this was in fact dead code and has been removed.